core/controlplane/config/templates/cluster.yaml

```diff
@@ -1309,6 +1309,13 @@ experimental:
   # Must be N times more than kubelet's nodeStatusUpdateFrequency (default 10s).
 #   nodeMonitorGracePeriod: "40s"
 
+kubelet:
+  # Tell Kubelet to renew its certificate as its expiration time is approaching
+  # Requires FeatureGate RotateKubeletClientCertificate=true on worker
```

Comment:
You say this feature gate is required, but why not adding that feature gate automatically if the certificate rotation is enabled?

Reply:
It could be done, Is there any convention about that?

Reply:
I don't believe there's a convention for this, only that we (ideally) should avoid forcing the user to change two places (in this case, `cluster.yaml` and  `userdata/cloud-config-worker`) in order to enable any given feature.

Reply:
FeatureGates can be added to the cluster.yaml (and this is what i meant in the comment)

```
worker:
  featureGate:
     RotateKubeletClientCertificate: true
```

I'll check how to do it automatically though

Reply:
Ah, I did not know about this flag in `cluster.yaml`. 😅 

Sorry for the confusion.

Reply:
After reviewing the code, I've seen that automatic FeatureGate enabling is being done for another component:

See https://github.com/kubernetes-incubator/kube-aws/blob/master/core/nodepool/config/config.go#L291

Reply:
Yep it definitely makes sense to me!
Would you mind doing the same for this feature, too?